### PR TITLE
First steps to implement translation support

### PR DESCRIPTION
**Pull request by riqueenz**
Added the files needed to translate the first sentence. Added a translation test phrase in the python command line that was successfully translated. The implementation is based on the translation of the A2Plus workbench.

### CODE REVIEW

File: translations/fasteners_pt-br.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE TS>
+<TS version="2.1" language="pt_BR" sourcelanguage="en_US">
+<context>
+    <name>InitGui</name>
+    <message>
+        <location filename="a2plib.py" line="1130"/>
```
Comment (luzpaz) [inline translations/fasteners_pt-br.ts:7]: a2plib.py ?

Comment (riqueenz) [reply translations/fasteners_pt-br.ts:7]: I didn't use an automated script to create the .ts file. So I copied the .ts file from the A2Plus bench and edited it manually and forgot to change the file location reference. The translation worked anyway. In the future I will add a script to generate the .ts files automatically and this issue will no longer occur.

Comment (riqueenz) [reply translations/fasteners_pt-br.ts]: The location filename has been fixed.

**Comment by shaise** [comment]
Great idea, thanks!

**Comment by luzpaz** [comment]
Thank you @riqueenz. Are you planning to write documentation for this?

**Comment by riqueenz** [comment]
> Thank you @riqueenz. Are you planning to write documentation for this?

Initially, the priority is to translate this workbench. Maybe I'll make a very simplified documentation and improve it when the translation is complete.

**Pull request merged**